portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -1306,19 +1307,30 @@ public DLFileEntry getFileEntryByUuidAndGroupId(String uuid, long groupId)
 
 	@Override
 	public Map<Long, Long> getFileEntryTypeIds(
-		long companyId, long groupId, String treePath) {
+		long companyId, Long[] groupIds, String treePath) {
```

Comment:
Why is this a `Long[]` instead of a `long[]`?

Reply:
`in(`needs the Object and to do not do the `ArrayUtil.toLongArray(` here.
but I can do it here if is more clear.